Geometry/VeryForwardGeometry/python/geometryRP_cfi.py

```diff
@@ -81,8 +81,38 @@
         'Geometry/VeryForwardData/data/RP_Cuts_Per_Region.xml',
         'Geometry/VeryForwardData/data/RP_Param_Beam_Region.xml')
 
+ctppsDiamondGeomXMLFiles = cms.vstring(
```

Comment:
OK. I wasn't sure if these changes were bugfixes or an affect of some other refactoring unrelated to CTPPS